plugins/module_utils/lxd.py

```diff
@@ -80,6 +80,8 @@ def do(self, method, url, body_json=None, ok_error_codes=None, timeout=None):
         if resp_json['type'] == 'async':
             url = '{0}/wait'.format(resp_json['operation'])
             resp_json = self._send_request('GET', url)
+            while resp_json['metadata']['status'] == 'Running':
```

Comment:
This looks like very specific logic inside a very generic function. There should at least be a parameter for this function which enables or disables this code, so it is only used in the specific cases where this feature should work.

Reply:
Added